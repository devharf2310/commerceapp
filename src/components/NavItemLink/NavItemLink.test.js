import React from 'react'
import renderer from 'react-test-renderer'
import MockRouter from 'react-mock-router';
import NavLink from './NavItemLink'

const props = {
    exact: true,
    label: "All",
    to: "/products",
    icon: "format_list_bulleted"
}


describe('Nav Item Link snapshot tests', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<MockRouter><NavLink {...props} key={props.to} /></MockRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
