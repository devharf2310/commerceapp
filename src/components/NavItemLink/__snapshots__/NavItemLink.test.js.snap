// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Item Link snapshot tests renders correctly 1`] = `
<li
  className="md-list-item"
  style={Object {}}
>
  <a
    aria-expanded={null}
    aria-pressed={false}
    className="md-fake-btn md-pointer--hover md-fake-btn--no-outline md-list-tile md-list-tile--icon md-text"
    onBlur={[Function]}
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex={0}
  >
    <div
      className="md-ink-container"
    />
    <div
      className="md-tile-addon md-tile-addon--icon"
    >
      <i
        className="md-icon material-icons"
      >
        format_list_bulleted
      </i>
    </div>
    <div
      className="md-tile-content md-tile-content--left-icon"
    >
      <div
        className="md-tile-text--primary md-text"
      >
        All
      </div>
    </div>
  </a>
</li>
`;
